packages/ui/src/accounts/components/AccountItem/components/styles.ts

```diff
@@ -72,6 +72,7 @@ export const LocksButtons = styled.div`
   align-self: end;
   display: flex;
   column-gap: 10px;
+  align-items: center;
```

Comment:
I thought i wrote this before but could not find my comment:
This  line can be removed, has no impact at all.

Reply:
Had the same feeling. Likely because of commit overlap with #3828.